worker/src/durable.rs

```diff
@@ -311,6 +312,41 @@ impl Storage {
             .map_err(Error::from)
     }
 
+    /// Retrieves the current alarm time (if set) as integer milliseconds since epoch.
+    /// The alarm is considered to be set if it has not started, or if it has failed
+    /// and any retry has not begun. If no alarm is set, `get_alarm()` returns `None`.
+    pub async fn get_alarm(&self) -> Result<Option<i64>> {
```

Comment:
Should have an optional options parameter that matches [getAlarm](https://github.com/cloudflare/workers-types/blob/master/index.d.ts#L433) from the workers-types repo.